src/content_mapblock.cpp

```diff
@@ -172,6 +172,14 @@ static inline void getNeighborConnectingFace(v3s16 p, INodeDefManager *nodedef,
 		*neighbors |= v;
 }
 
+// For use in mapblock_mesh_generate_special
+// X,Y,Z of position must be -1,0,1
+// (If you don't know what this function does, you shouldn't be using it).
```

Comment:
That last line "(If you don't know what this function does, you shouldn't be using it)" is not appropriate. It's not about knowing what the function does, it's about documentation for future developers.